Documentation/cubes.rst

```diff
@@ -5,24 +5,67 @@ A ``Cube`` is a logical entity, composed by a ``data plane`` (which can include
 
 In addition, a Cube may include also a ``slow path`` in case some additional data plane-related code is required to overcome some limitations of the eBPF data plane.
 
-Following is an example including instances of Router and Bridge Cubes, and a possible topology that interconnects them.
+Types of Cubes
+--------------
+
+Polycube has two diferent kind of cubes, **standard cubes** and **transparent cubes**.
+
+Standard cubes have forwarding capabilities, such router and bridge.
+
+A standard cube:
+  - defines a :ref:`ports <cube-ports>` concept, ports are *connected* to other ports or netdevs
+  - makes a forwarding decision, i.e, sending the packet to another port
 
 ::
 
+             standard cube
+           +--------------+
+           |              |
+           | +----------+ |
+   port1---|-|          |-|---port3
+           | |   core   | |
+   port2---|-|          |-|---port3
+           | +----------+ |
+           |              |
+           +--------------+
 
-     veth1                                                             veth3
-       |                                                                |
-       |                                                                |
-       |                                                                |
-  +----------+        +----------+             +----------+        +----------+
-  |   br1    |        |    r1    |             |    r2    |        |   br2    |
-  |  bridge  |--------|  router  |-------------|  router  |--------|  bridge  |---------veth5
-  |  (cube)  |        |  (cube)  |             |  (cube)  |        |  (cube)  |
-  +---------.+        +----------+             +----------+        +----------+
-       |                                                                |
-       |                                                                |
-       |                                                                |
-     veth2                                                            veth4
+A trasparent cube is a cube without any forwarding capability, such as a network monitor, NAT and a firewall.
+
+A trasparent cube:
+  - does not define any port, but it *attaches* to existing ports such as the ones of a normal service (e.g., port1 on router2) or a network interface in the host (e.g., veth0);
+  - it *inherits* all the parameters associated to ports (e.g., MAC/IPv4 addresses, link speed, etc.) from the actual port it is attached to (e.g., port 1 on router2, or veth0);
+  - cannot be *connected* to other services, but has to be *attached* to an existing port.
```

Comment:
==>
(merge first and third item):
- does not define any port, hence cannot be *connected* to other services, but has to be *attached* to an existing port such as the one of a normal service (e.g., port1 on router2) or a network interface in the host (e.g., veth0);